components/TARGET_PSA/services/psa_prot_internal_storage/psa_prot_internal_storage.h

```diff
@@ -1,40 +1,49 @@
-/* Copyright (c) 2018 ARM Limited
- *
- * SPDX-License-Identifier: Apache-2.0
+#ifndef __INTERNAL_TRUSTED_STORAGE_H__
```

Comment:
Not a bug, but this should come after the license as it was in the file. not sure why changed